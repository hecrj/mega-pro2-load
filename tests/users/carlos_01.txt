4

1000
150
400
50

7

1 2 4 0 0 5 0 0 3 6 0 0 7 0 0

50
4 1 2 3 4
50
4 1 2 3 4
50
4 1 2 3 4
50
4 1 2 3 4
50
4 1 2 3 4
50
4 1 2 3 4
50
4 1 2 3 4

-1 2 50

-3 51

-1 1 55

-3 56

-1 1 57

-1 1 58

-3 59

-1 1 60

-1 1 61

-2 62

-3 63

-5 49 51

-5 49 70

-2 100

-1 3 101

-1 4 102

-1 2 103



-6

