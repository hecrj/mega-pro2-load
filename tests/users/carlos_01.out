Peticion procesada y servidores
0 1
1 2 4
Servidores ocupados
0
Peticion procesada y servidores
1 7
1 2 4
Servidores ocupados
1 1 6
2 1 6
4 1 6
Peticion procesada y servidores
2 10
3 6
Peticion procesada y servidores
3 20
5
Servidores ocupados
1 1 3
2 1 3
3 2 8
4 1 3
5 3 19
6 2 8
Peticion procesada y servidores
4 20
7
Peticion procesada y servidores
5 0
Peticiones pendientes
2 1 57 67
3 1 58 78
4 1 60 80
Servidores ocupados
3 2 4
5 3 15
6 2 4
7 4 17
Pelicula mas solicitada
2 1
Pelicula mas solicitada
1 4
Peticiones pendientes
0
Peticion procesada y servidores
6 3
1 2 4
Peticion procesada y servidores
7 1
3
Peticion procesada y servidores
8 2
3 6
